Formula/libvoikko.rb

```diff
@@ -1,9 +1,8 @@
 class Libvoikko < Formula
   desc "Linguistic software and Finnish dictionary"
   homepage "https://voikko.puimula.org/"
-  url "https://www.puimula.org/voikko-sources/libvoikko/libvoikko-4.3.tar.gz"
-  sha256 "e843df002fcea2a90609d87e4d6c28f8a0e23332d3b42979ab1793e18f839307"
-  revision 3
+  url "https://www.puimula.org/voikko-sources/libvoikko/libvoikko-4.3.1.tar.gz"
+  sha256 "368240d4cfa472c2e2c43dc04b63e6464a3e6d282045848f420d0f7a6eb09a13"
```

Comment:
```suggestion
  sha256 "368240d4cfa472c2e2c43dc04b63e6464a3e6d282045848f420d0f7a6eb09a13"
  license "GPL-2.0-only"
```
From `COPYING`:
```
The following is the license under which libvoikko as a whole can be used.
Parts of the library are additionally available under other licenses. See
LICENSE.CORE for more information on those licenses.


		    GNU GENERAL PUBLIC LICENSE
		       Version 2, June 1991
```
Very few of the source headers say "at your option", and the ones that do are in the `test` directory, or are things like `aclocal.m4` or `config.sub`.